code/modules/clothing/suits/jobs.dm

```diff
@@ -259,6 +281,27 @@
 	blood_overlay_type = "coat"
 	body_parts_covered = UPPER_TORSO|ARMS
 
+//Head of Security
+/obj/item/clothing/suit/mantle/armor
+	name = "armored shawl"
+	desc = "A reinforced shawl, worn by the Head of Security. Do you dare take up their mantle?"
+	icon_state = "hosmantle"
+	item_state = "hosmantle"
+	allowed = list(/obj/item/gun/energy,/obj/item/reagent_containers/spray/pepper,/obj/item/gun/projectile,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/restraints/handcuffs,/obj/item/flashlight/seclite,/obj/item/melee/classic_baton/telescopic,/obj/item/kitchen/knife/combat)
+	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0)
+	heat_protection = UPPER_TORSO|ARMS
+	max_heat_protection_temperature = ARMOR_MAX_TEMP_PROTECT
+	strip_delay = 60
+	put_on_delay = 40
+	burn_state = FIRE_PROOF
+
+//Head of Personnel
+/obj/item/clothing/suit/mantle/armor/head_of_personnel

```

Comment:
huge armor buff for the HoP.

the armor should be: `armor = list(melee = 25, bullet = 15, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0)`